packages/remix-node/package.json

```diff
@@ -12,7 +12,16 @@
   },
   "license": "MIT",
   "main": "dist/index.js",
+  "module": "dist/esm/index.mjs",
   "typings": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/esm/index.mjs",
```

Comment:
Since we're no longer re-exporting the `react-router/server` API, consumers import from both `react-router/server` and `@react-router/node`. To avoid the dual package hazard for CJS and ESM versions of the package, we need to ship both versions for this package too to ensure consumers get CJS + CJS or ESM + ESM and not a mix of the two. The alternative would be for us to remove the ESM version of `react-router/server` and only use CJS (which `@react-router/node` currently does).

This issue was caught because we're doing an `instanceof MaxPartSizeExceededError` check in `error-test` but the app actually has two copies of the `MaxPartSizeExceededError` class in memory from the CJS and ESM builds. The test failed until this additional ESM version of `@react-router/node` was added.